### DISCUSSION TITLE
Is it possible to add code128 barcode in the online designer ?

### DISCUSSION THREAD

**Discussion by stefaanMLB** [pdfme/pdfme#602] [Q&A]
During discovery of the online designer at https://pdfme.com/template-design, I wasn't able to add a code-128 barcode.
Only QR-code seems available.
Am I missing something ?

**Comment by peteward** [comment]
This page is just an example of what the pdfme template designer can do.
There are many other barcodes that you can add in your application. See [here](https://github.com/pdfme/pdfme/blob/main/playground/src/helper.ts#L113C1-L137C1) an example using a lot more field types and plugins, including a code-128 barcode.
